MeshLib/Mesh.cpp

```diff
@@ -269,5 +269,15 @@ void Mesh::setNodesConnectedByElements()
 	}
 }
 
+std::size_t Mesh:: getMaximumNConnectedNodesToNode() const
+{
+	std::size_t max_ncn = 0;
+	for(auto node : _nodes)
+	{
+		if(node->getConnectedNodes().size() > max_ncn)
+			max_ncn = node->getConnectedNodes().size();
+	}
+	return max_ncn + 1;
+}
```

Comment:
Yes,  I need it for the pre-memory allocation for PETSc matrix. Its usage will be shown in  one coming PR.